host/sgx/cpuid.h

```diff
@@ -4,12 +4,45 @@
 #ifndef _OE_CPUIDCOUNT_H
 #define _OE_CPUIDCOUNT_H
 
-void oe_get_cpuid(
+#if defined(__GNUC__)
+#include <cpuid.h>
+/* Same as __get_cpuid, but sub-leaf can be specified.
+   Need this function as cpuid level 4 needs the sub-leaf to be specified in ECX
+*/
+static inline void oe_get_cpuid(
     unsigned int __leaf,
     unsigned int __subleaf,
     unsigned int* __eax,
     unsigned int* __ebx,
     unsigned int* __ecx,
-    unsigned int* __edx);
+    unsigned int* __edx)
+{
+    __cpuid_count(__leaf, __subleaf, *__eax, *__ebx, *__ecx, *__edx);
+}
+/* endif defined(__GNUC__) */
+#elif defined(_MSC_VER)
+#include <intrin.h>
+#include <limits.h>
+/* Same as __get_cpuid, but sub-leaf can be specified.
```

Comment:
Nit: reduce repetition of function signature and comment by nesting the `ifdef` in the body of function.

Reply:
nice catch, thanks! addressed.